src/mongocxx/pool.cpp

```diff
@@ -39,20 +39,20 @@ void pool::_release(client* client) {
 
 pool::~pool() = default;
 
-pool::pool(const uri& uri, stdx::optional<options::ssl> ssl_options)
+pool::pool(const uri& uri, const options::pool& options)
```

Comment:
Right.  I wish GH could be taught to put .hpp files before .cpp files.  :-)

LGTM.